Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -88,9 +88,6 @@ final class PostCampaignCheckoutViewController: UIViewController {
   // MARK: - Configuration
 
   private func configureChildViewControllers() {
-    _ = (self.rootScrollView, self.view)
-      |> ksr_addSubviewToParent()
-
```

Comment:
We don't need this anymore because rootScrollView is being added to the view in the line below